src/lephare/data_manager.py

```diff
@@ -43,33 +42,78 @@ def configure_directories(self):
             )
 
         self.lephare_work_dir = os.getenv("LEPHAREWORK", None)
-        # If not set then set to default and make directories if not present
+
+        # If LEPHAREWORK is not set then set to default and make directories
         if self.lephare_work_dir is None:
-            # Default location in system cache
-            self.lephare_work_dir = this_os_cache + "/work"
-            # Set environment variable to default
-            os.environ["LEPHAREWORK"] = self.lephare_work_dir
-            if os.path.isdir(self.lephare_work_dir):
-                print(
-                    f"""Lephare default working directory already exists at
-                    {self.lephare_work_dir}."""
-                )
-            else:
-                # Make a timestamped 'run' directory which is symlinked from default
-                now = datetime.datetime.now().strftime("%Y%m%dT%H%M%S")
-                run_directory = f"{this_os_cache}/runs/{now}"
-                print(
-                    f"""Lephare default working directory being created at
-                    {self.lephare_work_dir}."""
-                )
-                os.makedirs(run_directory)
-                os.symlink(run_directory, self.lephare_work_dir)
+            # get <default_cache> locations
+            default_os_cache = user_cache_dir("lephare", ensure_exists=True)
 
-                work_sub_directories = ["filt", "lib_bin", "lib_mag", "zphota"]
-                for sub_dir in work_sub_directories:
-                    os.makedirs(os.path.join(self.lephare_work_dir, sub_dir))
+            # create a runs directory in the default cache locations
+            os.makedirs(f"{default_os_cache}/runs", exist_ok=True)
+
+            # create a timestamped directory under runs
+            now = datetime.datetime.now().strftime("%Y%m%dT%H%M%S")
```

Comment:
(Just to better understand/for personal curiosity, this timestamped run directory is made so we'll have something to build a <user cache>/work symlink to, so our work directory will look right for create_new_run, right? Or does all output get send to the current "work" dir until a user chooses to manually call create_new_run?)

Either way, it looks like this bit of code overlaps some with 100-112 - it should run totally fine, but might want to make a helper to avoid duplication

Reply:
Good questions - the intent of the work directory is to always be a pointer to the most recently created timestamped run directory. From Lephare's perspective, it will attempt to write output to `.../work/filt` (for instance) but that will actually be writing to `.../runs/<latest_timestamp>/filt`. 

If a user then wants to go back to a previous run and with a new config file for instance, they can then set the environmental variable `LEPHAREWORK=.../runs/<previous_timestamp>` and expect Lephare to look at this directories specifically.

Reply:
And yes, very good eye, there is a bit of code overlap between this section and 100-112, initially I had these merged into a subfunction but I was having quite a bit of difficulty keeping the different use cases working when I was writing tests. I ended up duplicating some code to make it easier to get the tests into a working state, but now that they are, I can create a followup issues to try to merge the logic back together.

Reply:
Captured that here: https://github.com/lephare-photoz/lephare/issues/77